src/store/reducers/dform/initial/styles.json

```diff
@@ -2976,6 +2976,72 @@
             }
           ]
         },
+        {
+          "class": "block",
+          "label": "WIDEL TEST",
+          "children":[
+            {
+              "field": "widelTest.positive",
+              "label": "Positive",
+              "class": "check"
+            },
+            {
+              "field": "widelTest.negative",
+              "label": "Negative",
+              "class": "check"
+            }
+          ]
+        },
+        {
+          "class": "block",
+          "show": "widelTest.positive",
+          "children":[
+            {
+              "field":"widelTest.Positive_SalO",
+              "label": "Sal. Typhi O",
+              "class": "radio",
+              "options":[
+                {
+                  "id": "< or = 1 : 80",
```

Comment:
While this works perfectly, but just for convention, I think ` ` (white space) should not be used in `"id"` field because it is used as a key of JSON structure.

Reply:
Ahh.. I will update it soon. Thank you..